CommonMark/CommonMarkAdditionalFeatures.cs

```diff
@@ -21,6 +21,26 @@ public enum CommonMarkAdditionalFeatures
         StrikethroughTilde = 1,
 
         /// <summary>
+        /// The parser will recognize syntax <c>~foo~</c> that will be rendered as <c>&lt;sub&gt;foo&lt;/sub&gt;</c>.
+        /// </summary>
+        SubscriptTilde = 2,
+
+        /// <summary>
+        /// The parser will recognize both <see cref="StrikethroughTilde"/> and <see cref="SubscriptTilde"/>.
+        /// </summary>
+        Tilde = 3,
+
+        /// <summary>
+        /// The parser will recognize syntax <c>^foo^</c> that will be rendered as <c>&lt;sup&gt;foo&lt;/sup&gt;</c>.
+        /// </summary>
+        SuperscriptCaret = 4,
+
+        /// <summary>
+        /// The parser will treat reference labels as case sensitive.
+        /// </summary>
+        RespectReferenceCase = 0x100,
```

Comment:
well, this does not really work for one as me whoe read the value as binary for some reason and hex so it looked like `8`. Why do you want a distant value? And you can always add `//comment` after the value - that might be better so that the next contributor know whether to use 8 or 512.